base/services/telnetd/telnetd.c

```diff
@@ -367,12 +367,19 @@ static void RunShell(client_t *client)
    STARTUPINFO           si;
    PROCESS_INFORMATION   piProcInfo;
    SECURITY_ATTRIBUTES   saAttr;
+   char cmd_path[MAX_PATH];
 
-   const char *name = "c:\\reactos\\system32\\cmd.exe";
+   //const char *name = "c:\\reactos\\system32\\cmd.exe";
```

Comment:
I assume this line is meant to be fully removed.

Reply:
OK.